_posts/blog/http/2018-11-08-napetitions-review.md

```diff
@@ -0,0 +1,51 @@
+
+Take, for example, the 1762 petition of Esther Anthony, a Native woman of color from the tribe of Mashpee, but living in the town of Middleborough, Massachusetts. The [informational page](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/W0R4MJ) includes several identifiers, including the total number of signatures, the extent of legislative action, the number of females of color signatures, tribal affiliation, and additional archivist notes (i.e., widow, Jeremiah Anthony, sale, lands, payments). At the top of the page is a full citation of the original primary source along with the petition subject (which, in this case is property) and a link to a [high resolution photograph](https://iiif.lib.harvard.edu/manifests/view/drs:422116544$140i) of the petition. In the Harvard Library viewer, users can "flip" through the selected petition or browse related petitions (left of the screen) either submitted on the same date, on similar topics, or by the same petitioners as they would be organized in an archival folder. This user-friendly layout provides ample room for researchers interested in larger historical trends, genealogical networks, or public history projects. The latter opens the door to tellings of more nuanced narratives, powerful stories, specific individuals names, and the movement of ideas. Dr. Carpenter demonstrates one such application in a data visualization project called [“Your Humble Petitioners”.](http://www.yourhumblepetitioners.com/mapview/antislaveryma) This interactive map associates geographic coordinates with the locations inscribed in the metadata of the antislavery petitions. This could very easily be replicated with the Native American petitions. 
+
+**Conclusion**   
+The priority to distinguish Native petitions, or petitions about Native issues, commits to a partial restructure the power balance of the colonial archive. However, one is forced to ask whether it goes far enough. Is attribution enough?  “Authorship is foundational to the organizational framework because it operates as a core node in the classification of all materials within the modern archive”(Anderson, 230). How is authorship determined? Is that the role of the archivist or the researcher? While this archive does not answer those questions, it does open up a world of research into key figures and names in the history of Indigenous petitioning efforts in the Northeast and is a valuable resource to an interdisciplinary and intertribal pool of scholars.    
```

Comment:
This paragraph reiterates what you say in the introduction. I'd like you to go farther by giving an answer, even if it's only a partial one. "The project shows us how authorship can do [x]... but leaves [y] unresolved."